salt/loader.py

```diff
@@ -1629,7 +1629,11 @@ def _load(self, key):
                 return True
             # if the modulename isn't in the whitelist, don't bother
             if self.whitelist and mod_name not in self.whitelist:
-                raise KeyError
+                log.error(
+                    'Failed to load key %s because its module (%s) is not in '
```

Comment:
If someone reading the log won't start panicking about SSH pr SSL keys... Can we rephrase this to something else?

Reply:
Sure, that is a good point.

Reply:
Done.